content/about/introduction/introduction.html

```diff
@@ -14,166 +14,161 @@
+          APGのアクセシビリティガイダンスは、<a href="https://www.w3.org/TR/WCAG21/">WCAG</a>および<a class="specref" href="#abstract">ARIA</a>によって指定されたアクセシビリティ要件とは異なります。
+          WCAGとARIAは「W3C規範的技術標準」であり、APGは「情報提供」リソースです。
+          「W3Cの勧告」と公式に知られているW3C規範的技術標準には、標準で文書化された要件に準拠する方法を定義する適合モデルがあります。
+          これらの要件は、<a href="https://www.w3.org/Consortium/Process#rec-track">W3C Recommendation Track</a>によって定義された厳格なプロセスを使用して開発されます。
         </p>
         <p>
-          The APG does not specify normative requirements and thus does not have a conformance model.
-          The <a href="https://www.w3.org/groups/tf/aria-practices">ARIA Authoring Practices Task Force</a>, which is part of the <a href="https://www.w3.org/groups/wg/aria">ARIA Working Group</a>,
-          uses consensus to ensure APG guidance is consistent with all relevant specifications, including ARIA, but its process for doing so is not bound by the W3C Recommendation Track process.
+          APGは規範的な要件を指定せず、したがって適合モデルを持っていません。
+          <a href="https://www.w3.org/groups/tf/aria-practices">ARIA Authoring Practices Task Force</a>は、<a href="https://www.w3.org/groups/wg/aria">ARIA Working Group</a>の一部であり、APGガイダンスがARIAを含むすべての関連仕様と一致していることを確実にするために合意を使用しますが、そのプロセスはW3Cの勧告トラックプロセスによって束縛されません。
         </p>
         <p>
-          Some of the guidance in the APG is not covered by a normative specification.
-          For example, the patterns include keyboard interaction models that combine a normative requirement specified by ARIA with conventions established by popular operating systems and software.
-          In the section on <a class="specref" href="#managingfocus">Managing focus</a>, ARIA includes normative language stating that the <kbd>Tab</kbd> key should not move focus inside of certain types of widgets.
-          However, ARIA does not specify how specific keys should move focus inside of those widgets.
-          So, the APG Task Force relies on broadly utilized keyboard conventions to provide each pattern with a list of keyboard commands for navigation and operation.
-          While those lists are not normative, the majority of such conventions are so well established that it is advisable to treat them as requirements.
-          Where that is not the case, the guidance is clearly marked as optional.
-          Note that there are some keyboard commands where users could be better served if a web widget were to dynamically adjust to mirror the users' operating system conventions.
-          It is a <a href="https://github.com/w3c/aria-practices/issues/1438">known issue</a> that some APG keyboard interaction models do not yet provide sufficient macOS-specific guidance.        </p>
+          APGのガイダンスの一部は規範的な仕様によってカバーされていません。
+          例えば、パターンには、ARIAによって指定された規範的な要件と、一般的なオペレーティングシステムおよびソフトウェアによって確立された規約を組み合わせたキーボードインタラクションモデルが含まれています。
+          <a class="specref" href="#managingfocus">フォーカスの管理</a>のセクションでは、ARIAには特定のタイプのウィジェット内で<kbd>Tab</kbd>キーがフォーカスを移動させないようにするという規範的な言語が含まれています。
```

Comment:
normative language は、規範的な文言とか、規範的な記述、あたりが適切な訳かなあと思いましたがいかがでしょうか。

```suggestion
          <a class="specref" href="#managingfocus">フォーカスの管理</a>のセクションでは、ARIAには特定のタイプのウィジェット内で<kbd>Tab</kbd>キーがフォーカスを移動させないようにするという規範的な記述が含まれています。
```